assets/differentialabundance_report.Rmd

```diff
@@ -178,13 +194,44 @@ make_params_table <- function(name, pattern = NULL, remove_pattern = FALSE){
 }
 
 report_title <- paste0('Differential ',  params$features_type, ' abundance report', ifelse(is.null(params$report_title), '', paste0(': ', params$report_title)))
-report_subtitle <- paste0(ifelse(is.null(params$report_author), '', paste0('By ', params$report_author, ', ')), 'differentialabundance workflow version', versions[["Workflow.nf-core/differentialabundance"]])
+report_subtitle <- paste0(ifelse(is.null(params$report_author), '', paste0('By ', params$report_author, ', ')), '<br>differentialabundance workflow version', versions[["Workflow.nf-core/differentialabundance"]])
 ```
 
 ---
 title:  "<img src=\"`r file.path(params$input_dir, params$logo)`\" style=\"float: left;\"/>`r report_title`"
 subtitle: `r report_subtitle`
 ---
+\
+
+```{r, results='asis', echo=F, eval=T}
+if (!is.null(params$report_pi_name)) {
+  cat(paste0("**Project Members**:  \n"))
+  cat(paste0((params$report_pi_name)))
+}
+# Add further PI info only if name was provided
```

Comment:
Let's just have strings that we split containing contributors, roles etc. I don't think we need to hard-code the concept of a 'PI'.